src/pyfmi/fmi3.pyx

```diff
@@ -101,19 +101,24 @@ cdef void importlogger3(FMIL.jm_callbacks* c, FMIL.jm_string module, FMIL.jm_log
 
 cdef class FMI3ModelVariable:
     """ Class defining data structure based on the XML elements of ModelVariables. """
-    def __init__(self, name, value_reference, data_type, description, variability, causality, initial):
+    def __init__(self, name, value_reference, data_type, description, variability, causality, alias, initial):
         self._name            = name
         self._value_reference = value_reference
         self._type            = data_type
         self._description     = description
         self._variability     = variability
         self._causality       = causality
         self._initial         = initial
+        self._alias           = alias
 
     def _get_name(self):
         return self._name
     name = property(_get_name)
 
+    def _get_alias(self):
+        return self._alias
+    alias = property(_get_alias)
```

Comment:
Should we rename this to `is_alias` considering its a bool?

Reply:
I'd say no for consistency with older FMI versions.

Reply:
I would argue that this is the time to change it if we want to. But since this is not "game breaking" I'm OK with it, feel free to resolve. I agree that consistency is good to maintain where possible.